docs/home/howto/modules.md

```diff
@@ -0,0 +1,429 @@
+                            operator: In
+                            values:
+                              - pgpool
+                      topologyKey: kubernetes.io/hostname
+            nodeSelector: {}
+            tolerations: {}
+          resources:
+            limits:
+              memory: 176Mi
+            requests:
+              cpu: 250m
+              memory: 176Mi
+          pgpool:
+            env:
+              PGPOOL_BACKEND_NODES: autoconfigured
+              PGPOOL_POSTGRES_USERNAME: epi_pgpool_postgres_admin
+              PGPOOL_SR_CHECK_USER: epi_pgpool_sr_check
+              PGPOOL_ADMIN_USERNAME: epi_pgpool_admin
+              PGPOOL_ENABLE_LOAD_BALANCING: true
+              PGPOOL_MAX_POOL: 4
+              PGPOOL_POSTGRES_PASSWORD_FILE: /opt/bitnami/pgpool/secrets/pgpool_postgres_password
+              PGPOOL_SR_CHECK_PASSWORD_FILE: /opt/bitnami/pgpool/secrets/pgpool_sr_check_password
+              PGPOOL_ADMIN_PASSWORD_FILE: /opt/bitnami/pgpool/secrets/pgpool_admin_password
+            secrets:
+              pgpool_postgres_password: PASSWORD_TO_CHANGE
+              pgpool_sr_check_password: PASSWORD_TO_CHANGE
+              pgpool_admin_password: PASSWORD_TO_CHANGE
+            pgpool_conf_content_to_append: |
+              #------------------------------------------------------------------------------
+              # CUSTOM SETTINGS (appended by Epiphany to override defaults)
+              #------------------------------------------------------------------------------
+              # num_init_children = 32
+              connection_life_time = 900
+              reserved_connections = 1
+            pool_hba_conf: autoconfigured
+        - name: pgbouncer
+          enabled: false
+          image_path: brainsam/pgbouncer:1.12
+          init_image_path: bitnami/pgpool:4.1.1-debian-10-r29
+          use_local_image_registry: false
+          namespace: postgres-pool
+          service:
+            name: pgbouncer
+            port: 5432
+          replicas: 2
+          resources:
+            requests:
+              cpu: 250m
+              memory: 128Mi
+            limits:
+              cpu: 500m
+              memory: 128Mi
+          pgbouncer:
+            env:
+              DB_HOST: pgpool.postgres-pool.svc.cluster.local
+              DB_LISTEN_PORT: 5432
+              LISTEN_ADDR: "*"
+              LISTEN_PORT: 5432
+              AUTH_FILE: "/etc/pgbouncer/auth/users.txt"
+              AUTH_TYPE: md5
+              MAX_CLIENT_CONN: 150
+              DEFAULT_POOL_SIZE: 25
+              RESERVE_POOL_SIZE: 25
+              POOL_MODE: transaction
+    provider: any
+  ```
+* Run `epicli` tool to install epiphany:
+  ```shell
+  epicli --auto-approve apply --file='/tmp/shared/epi.yml' --vault-password='secret'
+  ```
+  This will install PosgreSQL on one of machines and configure pgbouncer, pgpool and additional services to manage database connections. 
```

Comment:
```suggestion
  This will install PosgreSQL on one of machines and configure pgbouncer, pgpool and additional services to manage database connections. Please make sure you disable applications that you don't need. Also you can enable standard Epiphany services like kafka or rabbitmq, by increasing number of virtual machines in the basic infrastructure config and assigning them to Epiphany components you want to use.
  If you would like to deploy custom resources into managed Kubernetes, then standard kubeconfig yaml document can be found inside the shared state file (you should be able to use vendor tools as well to get it).  
  We highly recommend using the `Ingress` resource in Kubernetes to allow access to web applications inside the cluster. Since it's managed Kubernetes and fully supported by the cloud platform, the classic HAProxy load-balancer solution seems to be deprecated here.
```